PyRDF/backend/Utils.py

```diff
@@ -1,10 +1,41 @@
-import ROOT
-import os
 import logging
+import multiprocessing
```

Comment:
Is this import required?

Reply:
it is to get the number of logical cpus in the machine, which I use in this implementation as an educated guess of how many jobs can be submitted concurrently from the Spark driver

Reply:
This is deprecated in favor of a simpler default number of threads = 4